.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: php
+php:
+- '7.2'
+- '7.0'
+
+install:
+  - composer install
+  - yarn
+
+before_script:
+  - touch database/database.sqlite
+  - cp .env.example .env
+  - php artisan key:generate
+  - php artisan migrate
```

Comment:
Wird nicht benötigt für die Tests. Die machen das mittels `use RefreshDatabase;` wo nötig (im Moment nicht nötig).

Reply:
jepp. env wird benötigt.

Reply:
key:generate logischerweise auch. da war ich ein bisschen zu schnell. we are gettingt there. I can feel it.

Reply:
Mein Kommentar bezog sich nur auf `php artisan migrate`. Das wird von github etwas missverständlich dargestellt.